### ISSUE TITLE
Could not determine the dependencies of task ':prepareSandbox'.

### ISSUE THREAD

**Issue by aishang5wpj** [JetBrains/gradle-intellij-plugin#898]
Hi, I created an plugin for Android Studio, use 'intellij.localPath' to specify AS path, but it shows error to me when I excuted runIde task.

Here is the error msg:

```
Could not determine the dependencies of task ':prepareSandbox'.
> Failed to query the value of task ':prepareSandbox' property 'pluginJar'.
   > The value of a manifest attribute must not be null (Key=Build-SDK).

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

Here is my config:

```
//build.gradle 

plugins {
    id 'java'
    id 'org.jetbrains.intellij' version '1.3.1'
    id 'org.jetbrains.kotlin.jvm' version '1.5.0'
}

group 'org.example'
version '1.0-SNAPSHOT'

repositories {
    mavenCentral()
}

dependencies {
    implementation "org.jetbrains.kotlin:kotlin-stdlib"
    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
}

// See https://github.com/JetBrains/gradle-intellij-plugin/
intellij {
    // Require the Android plugin, Gradle will match the plugin version to intellij.version
    plugins = ['android']

    localPath = '/Applications/Android Studio.app'
}

patchPluginXml {
    changeNotes = """
      Add change notes here.<br>
      <em>most HTML tags may be used</em>"""
}
test {
    useJUnitPlatform()
}
```

When I replace  'intellij.localPath' with  'intellij.version' it works good.

```
intellij {
    version = '203.7717.56'
    type = 'IC'              // Use IntelliJ IDEA CE because it's the basis of the IntelliJ Platform
    // Require the Android plugin, Gradle will match the plugin version to intellij.version
    plugins = ['android']
}
``` 

Could you please tell me what's problem with it? Does 'intellij.localPath'  can not be applied to specify local Android Studio path？

**Comment by hsz** [comment]
@aishang5wpj Thank you for the detailed description. This issue duplicates #881 and was fixed with #880.
Fix will be available in `v1.4.0`

**Comment by aishang5wpj** [comment]
@hsz [hsz](https://github.com/hsz) Okay, I get it!
